### ISSUE TITLE
Product Query: Create `All Products` variation

### ISSUE THREAD

**Issue by gigitux** [woocommerce/woocommerce-gutenberg-products-block#6787]
This issue is blocked by woocommerce/woocommerce-blocks#6795.

The `All Products` variation has to show all the products, with no further settings besides ordering. This version of the block won't work in tandem with filters.

**Comment by nerrad** [comment]
I think we decide we aren't doing this now? The Products Block is a complete replacement as is?

**Comment by gigitux** [comment]
> I think we decide we aren't doing this now? The Products Block is a complete replacement as is?

Good point! I think that we can close this issue. What do you think, @sunyatasattva?

**Comment by sunyatasattva** [comment]
The original idea of this block was to have a “setting-less” block which would be easy to just add and avoid all sorts of overwhelming options for the users, making an easy replacement for the “All Products” block and hiding complexity for the very common use-case.

This is currently in the “Icebox” section of woocommerce/woocommerce#42616, I will also add an `On Hold` label. I think it's premature to close it as we might want to gather a bit of feedback on how the block is used and this might be worthwhile to explore an easy option.

If you folks decide it'd be better to close it, it's ok for me as well, of course.

**Comment by nerrad** [comment]
I think we should just close, we can always re-open or create a new issue if feedback comes in warranting a separate block. The description as is, isn't very informative either.
